core/chains/evm/headtracker/types/types.go

```diff
@@ -40,9 +41,7 @@ type HeadTracker interface {
 // after being subscribed to HeadBroadcaster
 //
 //go:generate mockery --quiet --name HeadTrackable --output ../mocks/ --case=underscore
-type HeadTrackable interface {
-	OnNewLongestChain(ctx context.Context, head *evmtypes.Head)
-}
+type HeadTrackable = types.HeadTrackable[*evmtypes.Head]
```

Comment:
Oh nice, I didn't realize this could be done without changing all the other Head subscribers!
Thanks!